tsconfig.json

```diff
@@ -6,7 +6,7 @@
     "allowSyntheticDefaultImports": true,
     "module": "esnext",
     "target": "es5",
-    "lib": ["es2019", "dom"],
+    "lib": ["es2020", "dom"],
```

Comment:
Need es2020 to use [Promise.allSettled](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/allSettled)

Reply:
Travis build is still unhappy for some reason.
TS2550: Property 'allSettled' does not exist on type 'PromiseConstructor'. Do you need to change your target library? Try changing the 'lib' compiler option to 'es2020' or later.
105       return Promise.allSettled(promises)

Reply:
Needed to update tsconfig.build.json